### PR TITLE
EZP-26699 : translate fielddefinition

### PR DESCRIPTION
**Pull request by nicolas-bastien**
Jira : https://jira.ez.no/browse/EZP-26699

## Description

Translate field definition blocks

## Next

After this PR is merge, todo : synchronize ezplatform-18n

### CODE REVIEW

File: eZ/Bundle/EzPublishCoreBundle/Resources/views/fielddefinition_settings.html.twig [hunk 1]
```diff
@@ -290,93 +293,93 @@
 
 {% block settings_maxfilesize %}
     <li class="ez-fielddefinition-setting maximum-file-size">
-        <div class="ez-fielddefinition-setting-name">Maximum file size:</div>
+        <div class="ez-fielddefinition-setting-name">{{ 'fielddefinition.maximum-file-size.label'|trans|desc("Maximum file size:")}}</div>
         <div class="ez-fielddefinition-setting-value">
         {% if fielddefinition.validatorConfiguration.FileSizeValidator.maxFileSize %}
             {# TODO l10n / unit #}
-            {{ fielddefinition.validatorConfiguration.FileSizeValidator.maxFileSize }} bytes
+            {{ fielddefinition.validatorConfiguration.FileSizeValidator.maxFileSize }} {{ 'fielddefinition.maximum-file-size.unit'|trans|desc("bytes")}}
```
Comment (bdunogier) [inline eZ/Bundle/EzPublishCoreBundle/Resources/views/fielddefinition_settings.html.twig]: The `bytes` part has been changed to `MB` in #1900 (beware, it's a gigantic PR), merged a couple minutes ago. Could you rebase on top of it and adapt ?

Also, I believe thazt the maxFileSize should be made a parameter of the string, could you change it ?

File: eZ/Bundle/EzPublishCoreBundle/Resources/views/fielddefinition_settings.html.twig [hunk 2]
```diff
@@ -290,93 +293,93 @@
...
 {% endblock %}
 
 {% block settings_preferredrows %}
     <li class="ez-fielddefinition-setting preferred-rows-number">
-        <div class="ez-fielddefinition-setting-name">Preferred number of rows:</div>
+        <div class="ez-fielddefinition-setting-name">{{ 'fielddefinition.preferred-rows-number.label'|trans|desc("Preferred number of rows:")}}</div>
         <div class="ez-fielddefinition-setting-value">
         {% if rows %}
-            {{ rows }} rows
+            {{ rows }} {{ 'fielddefinition.preferred-rows-number.unit'|trans|desc("rows")}}
```
Comment (bdunogier) [inline eZ/Bundle/EzPublishCoreBundle/Resources/views/fielddefinition_settings.html.twig]: Make `{{ rows }}` a parameter ?

File: eZ/Bundle/EzPublishCoreBundle/Resources/views/fielddefinition_settings.html.twig [hunk 3]
```diff
@@ -290,93 +293,93 @@
...
         </div>
     </li>
 {% endblock %}
 
 {% block settings_allowmultiple %}
     <li class="ez-fielddefinition-setting multiple">
-        <div class="ez-fielddefinition-setting-name">Allow multiple choices:</div>
-        <div class="ez-fielddefinition-setting-value">{{ isMultiple ? 'Yes' : 'No' }}</div>
+        <div class="ez-fielddefinition-setting-name">{{ 'fielddefinition.multiple.label'|trans|desc("Allow multiple choices:")}}</div>
+        <div class="ez-fielddefinition-setting-value">{{ isMultiple ? 'fielddefinition.multiple.yes'|trans|desc("Yes") : 'fielddefinition.multiple.no'|trans|desc("No") }}</div>
```
Comment (bdunogier) [inline eZ/Bundle/EzPublishCoreBundle/Resources/views/fielddefinition_settings.html.twig]: Wouldn't this be better with `transchoice` ?

**Comment by nicolas-bastien** [comment]
@bdunogier done + I rework the date part too

**Pull request merged**